### PR TITLE
Docs for tenants, tiers & companies

### PR DESCRIPTION
**Pull request by mattvagni**
Following changes:
- Fix 'first api call' code example which was missing in Typescript SDK tab
- Add 'away mode' in assignment page
- Added top level pages for Tenants
- Added top level pages for Tiers & SLAs
- Moved companies & tenants into the "Adding context" section
- Rephrased/re-positioned customer groups to not do 'tiers'
- Removed `_typename` from any example queries
- Fixed a few broken snippets unrelated to this.
- Added docs for tenants end-to-end.
- Added docs for tiers end-to-end.

### CODE REVIEW

File: api-reference/graphql/tenants/add-customers.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+You can add a customer to multiple tenants.
+
+When selecting the customer you can chose how to identify them. You can use the customer's email, externalId or id.
+
+For this mutation you need the following permissions:
+
+- `TODO`
+- `TODO`
```
Comment (mattvagni) [inline api-reference/graphql/tenants/add-customers.mdx:13]: TODO

File: api-reference/graphql/tenants/remove-customers.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+You can remove customers from multiple tenants in one API call.
+
+When selecting the customer you can chose how to identify them. You can use the customer's email, externalId or id.
+
+For this mutation you need the following permissions:
+
+- `TODO`
+- `TODO`
```
Comment (mattvagni) [inline api-reference/graphql/tenants/remove-customers.mdx:13]: TODO

File: api-reference/graphql/tenants/set-customer-tenants.mdx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: 'Set customer tenants'
+---
+
+You can also set all tenants for a customer. Unlike the more specific add or remove mutations this is useful if you are sycing tenants and customers with Plain.
+
+For this mutation you need the following permissions:
+
+- `TODO`
+- `TODO`
```
Comment (mattvagni) [inline api-reference/graphql/tenants/set-customer-tenants.mdx:10]: TODO

File: _snippets/graphql/update-customer-company.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+```graphql Mutation
+mutation updateCustomerCompany($customerId: ID!, $companyId: ID!, $companyDomainName: String!) {
+  updateCustomerCompany(
+    input: {
+      customerId: $customerId
+      companyIdentifier: { companyId: $companyId, companyDomainName: $companyDomainName }
```
Comment (mattvagni) [inline _snippets/graphql/update-customer-company.mdx:6]:
```suggestion
mutation updateCustomerCompany($customerId: ID!, $companyDomainName: String!) {
  updateCustomerCompany(
    input: {
      customerId: $customerId
      companyIdentifier: { companyDomainName: $companyDomainName }
```

File: _snippets/typescript-sdk/update-customer-company.mdx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+```tsx
+const res = await client.updateCustomerCompany({
+  customerId: 'c_01H14DFQ4PDYBH398J1E99TWSS',
+  companyIdentifier: {
+    companyId: 'co_01HRRMRPERZCK42HTPD4JQ57NB',
+    companyDomainName: 'nike.com',
+  },
```
Comment (mattvagni) [inline _snippets/typescript-sdk/update-customer-company.mdx:7]:
```suggestion
  companyIdentifier: {
    companyDomainName: 'nike.com',
  },
```

**Review by presmihaylov — APPROVED** [review]

**Pull request merged**